.azure/test-linux.yml

```diff
@@ -3,13 +3,14 @@ parameters:
     type: string
     displayName: "Version of Python to test"
 
+  - name: "testRust"
+    type: boolean
+
   - name: "testQPY"
     type: boolean
-    default: false
```

Comment:
We already were setting these values in all places, so this doesn't impact any call sites.

Here, I think it's better to be explicit and make the job author decide what behavior they want.

Reply:
Sounds fine to me - I originally added the default, but I don't feel strongly about it.